sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogVersionsSuite.scala

```diff
@@ -196,6 +196,8 @@ class HiveExternalCatalogVersionsSuite extends SparkSubmitTestUtils {
       "--name", "HiveExternalCatalog backward compatibility test",
       "--master", "local[2]",
       "--conf", "spark.ui.enabled=false",
+      "--conf", "spark.sql.hive.metastore.version=1.2.1",
```

Comment:
The default value of `spark.sql.hive.metastore.jars` is 1.2.1. Why you need to add this config?
According to the description of this PR, there should be 2.3.4 ?